.github/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ Most importantly:
 - Include your test code or file(s).
   If large, please provide a link to a repository or [gist](https://gist.github.com).
 
-We'll discuss your proposed changes and how they relate to the overarching goals of Mocha, detailed below in [⚽️ About Project Goals](#⚽️-about-project-goals).
+We'll discuss your proposed changes and how they relate to the overarching goals of Mocha, detailed below in [⚽️ About Project Goals](#️-about-project-goals).
```

Comment:
The link was broken before, but the current change in this PR doesn't fix it. This suggestion should fix it.

```suggestion
We'll discuss your proposed changes and how they relate to the overarching goals of Mocha, detailed below in [⚽️ About Project Goals](#%EF%B8%8F-about-project-goals).
```